frontend/coprs_frontend/coprs/filters.py

```diff
@@ -167,7 +172,6 @@ def time_ago(time_in, until=None):
     diff = now - datetime.datetime.fromtimestamp(time_in)
     return humanize.naturaldelta(diff)
 
-
```

Comment:
A leftover blank line :-)

Reply:
hmm, I will fix this line. However, I do notice neighbouring filter methods in `filters.py` have double blanks between them.